go/vt/sqlparser/sql.y

```diff
@@ -1400,8 +1442,14 @@ with_admin_opt:
 
 // TODO: Implement IGNORE, REPLACE, VALUES, and TABLE
 create_query_expression:
-  base_select_no_cte order_by_opt limit_opt lock_opt
+  base_select_no_cte order_by_opt limit_opt lock_opt into_opt
```

Comment:
Is `into` valid for create query expressions? This looks like it's for `CREATE TABLE AS SELECT * FROM FOO` and I don't think `into_opt` would apply here, but let me know if I'm missing something.